x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/permission/RemoteClusterPermissions.java

```diff
@@ -70,19 +77,29 @@ public class RemoteClusterPermissions implements NamedWriteable, ToXContentObjec
     // package private non-final for testing
     static Map<TransportVersion, Set<String>> allowedRemoteClusterPermissions = Map.of(
```

Comment:
I'd consider this for later, but not now. This would loose the ability to gate multiple privileges per transport versions, which is/was intentional.  I think we will need to apply similar logic to other cluster privileges as well as index privileges, so this will likely be refactored at some point.